src/Error/Error.php

```diff
@@ -198,7 +198,7 @@ private function guessTemplateInfo()
 
         $backtrace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS | DEBUG_BACKTRACE_PROVIDE_OBJECT);
         foreach ($backtrace as $trace) {
-            if (isset($trace['object']) && $trace['object'] instanceof Template && 'Twig_Template' !== \get_class($trace['object'])) {
+            if (isset($trace['object']) && $trace['object'] instanceof Template && 'Twig\Template' !== \get_class($trace['object'])) {
```

Comment:
Can't we have both depending on what the user is using?

Reply:
`get_class` always return the canonical class name, not the alias. So this was indeed forgotten when changing which name is the alias.

Reply:
hmmm, the code is actually correct. We are testing that the object is an instance of Template, but not of class Twig_Template.